modules/tracking/src/trackerKCF.cpp

```diff
@@ -87,18 +87,20 @@ namespace cv{
     * KCF functions and vars
     */
     void createHanningWindow(OutputArray _dst, const cv::Size winSize, const int type) const;
-    void inline fft2(const Mat src, std::vector<Mat> & dest) const;
-    void inline fft2(const Mat src, Mat & dest) const;
+    void inline fft2(const Mat src, std::vector<Mat> & _dest, std::vector<Mat> & _layers) const;
```

Comment:
Why some of the arguments starting with _ and others no? Are you using the leading _ to denote the arguments that are passed by reference?